business/istio_config.go

```diff
@@ -633,11 +632,27 @@ func (in *IstioConfigService) ParseJsonForCreate(resourceType, subresourceType s
 	default:
 		err = fmt.Errorf("object type not found: %v", resourceType)
 	}
+	if pivot != nil {
+		var marshalledBytes []byte
+
+		// Unmarshall, just to validate JSON sent by client.
+		err = json.Unmarshal(body, pivot)
+		if err == nil {
+			// Re-marshall, to get the cleaned JSON.
+			marshalledBytes, err = json.Marshal(pivot)
```

Comment:
Mmm I haven't tested locally. The re-marshall procedure would work in all the cases ?
I guess this should be also aligned with the json omitempty.

Also I guess if it may have some implication in the edit page of the config, I guess not, but worth to test as well.
I mean, once that Wizard is created, resources should be available in the Istio Config section and editable.

Reply:
I just did a small test on the edit page. Both VirtualServices and DestinationRules update correctly in the editor.